src/pybind/cephfs/cephfs.pyx

```diff
@@ -238,9 +239,9 @@ cdef make_ex(ret, msg):
     """
     ret = abs(ret)
     if ret in errno_to_exception:
-        return errno_to_exception[ret](msg)
+        return errno_to_exception[ret](ret, msg)
```

Comment:
how about adding the error code to msg here instead of adding it as another arg, e.g. using ```msg + (": error code %d" % ret)``` like the Error() construction below does already?

Passing multiple args to Exception.__init__ results in them being printed as a tuple:

```
>>> import rados
>>> print rados.Error(22, 'an error occurred')
(22, 'an error occurred')
```

which looks odd to an end user